### ISSUE TITLE
Please disable the activation of turbo buttons (select + any button) 

### ISSUE THREAD

**Issue by artemvodorez** [gamesir123/app#73]
IMPORTANT!!! Please disable the activation of turbo buttons (select + any button) or add the option to disable via the app.  it makes it very difficult to play.
GAMESIR X2

**Comment by diancabral** [comment]
I agree.. I'm trying to play Guitar Hero in PS2 emulator, but when i select the special with a button pressed, the notes start to fail. Please, put a option to disable this function in app.

**Comment by mynameislincks** [comment]
I agree, I didn't know this function even existed and it was driving me crazy while trying to play Super Metroid where Select is used to alternate weapons. I was thinking the control was bugged. Please make an option on the app to disable this function or remap to another trigger like the Home button (that I never use)

**Comment by rbreaves** [comment]
Yup.. like everyone is saying this needs to be able to be disabled entirely. It randomly activates itself & will undoubtedly result in many returns. Feature would be fine if it didn't randomly activate itself.

Also difficult to disable reliably. Who's testing this stuff?
